tests/runtime/http_server/test_http_server.py

```diff
@@ -457,6 +457,34 @@ def test_inference_streaming_sample_module(sample_task_model_id, runtime_http_se
         )
 
 
+def test_inference_streaming_sample_module_another_test(
```

Comment:
This is the exact same test as the `test_inference_streaming_sample_module`, but for some reason, this test hands indefinitely if they are both run together

Reply:
FYI the grpc test cases run perfectly fine, from my initial investigation, it seems like a bug with asyncio or sse, both of which target the http server